src/internal/connector/onedrive/collection.go

```diff
@@ -498,15 +498,60 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 				metaSuffix = metadata.DirMetaFileSuffix
 			}
 
-			// Fetch metadata for the file
 			itemMeta, itemMetaSize, err = oc.itemMetaReader(
 				ctx,
 				oc.service,
 				oc.driveID,
 				item)
 
 			if err != nil {
-				el.AddRecoverable(clues.Wrap(err, "getting item metadata").Label(fault.LabelForceNoBackupCreation))
+				// Get a fresh copy of item from graph and check if it's deleted
+				di, e := oc.itemGetter(
+					ctx,
+					oc.service,
+					oc.driveID,
+					ptr.Val(item.GetId()))
+
+				// Item is deleted if either of below conditions are true
+				// 1. Graph returns 404 not found
+				// 2. Item is not nil and has deleted property set
+				// Mark item as skipped and return
+				if clues.HasLabel(e, graph.LabelStatus(http.StatusNotFound)) ||
+					graph.IsErrDeletedInFlight(e) ||
+					(e == nil && di.GetDeleted() != nil) {
+					logger.CtxErr(ctx, err).
+						With("skipped_reason", fault.SkipNotFound).
+						Info("item not found")
+
+					skip := fault.ContainerSkip(
```

Comment:
We do not use fault skips for deleted items.  A fault.skip is only for tracking items that are valid, but which we are unable or unwilling to retrieve.  Malware is a good example: it's a valid file, we just don't want to deal with it (and msoft wont let us, anwyay).  A deleted/not-found item is not valid: it doesn't exist.

Reply:
The way we handle 404 is different across different services.I think we silently ignore them in Exchange, but we do add skips for them in OneDrive ([ref](https://github.com/alcionai/corso/blob/7181e2ef90eedcfe7451c9a96949a6978163a8c3/src/internal/connector/onedrive/collection.go#L347)). I believe what Keepers is suggesting is to silently ignore 404 as they are something that "does not exist" at the time of  backup and should be treated as if we never knew about them.

I think we should keep both Exchange and OneDrive consistent, but I'll let Keepers make the final call on what to do here.

Reply:
Yup!  Also, I forgot that we were "skipping" missing onedrive items.  We've had sync discussions stating the need to remove that skip and silently ignore it, instead.

Reply:
I'll remove the onedrive skip in a new PR